### PR TITLE
Add placeholder image and update instructional text

### PR DESCRIPTION
**Pull request by jehlers42**
I made a rough placeholder image to illustrate the display's pins in the hopes of reducing confusion. If it's good enough, perfect. If not, I'll get someone to work up a better one.

I also updated the display article to remove (hopefully) the ambiguity of the original text. The new image plays a role. The warning text is cribbed from the display's product brief.

The initial comment in issue #2977 makes it seem like you could technically power the Pi using the display by plugging into the display while the GPIOs are connected with jumpers, but I didn't include that as an option because it doesn't seem like a great one (if it's even possible).

Closes issues #2977 and #3007.

### CODE REVIEW

**Comment by aallan** [comment]
@jehlers42 Can you create an Asana task to get the placeholder image redone by Design when they get a chance. Thanks.

File: documentation/asciidoc/accessories/display/display_intro.adoc [hunk 1]
```diff
@@ -11,13 +11,28 @@ This whitepaper is an introduction to writing a Display Serial Interface (DSI) d
...
 
-=== Physical Installation
+=== Connecting a Raspberry Pi
 
-The following image shows how to attach the Raspberry Pi to the back of the Touch Display (if required), and how to connect both the data (ribbon cable) and power (red/black wires) from the Raspberry Pi to the display. If you are not attaching the Raspberry Pi to the back of the display, take extra care when attaching the ribbon cable to ensure it is the correct way round. The black and red power wires should be attached to the GND and 5v pins respectively.
+The following image shows how to attach a Raspberry Pi to the back of the Touch Display using the included standoffs. It also shows the connections you need to make between Raspberry Pi and the display's PCB. You can also mount the touch display in a separate chassis if you have one available. The connections remain the same, though you may need longer cables depending on the chassis you use.
 
-image::images/GPIO_power-500x333.jpg[DSI Display Connections]
+image::images/GPIO_power-500x333.jpg[DSI Display pinout]
```
Comment (aallan) [inline documentation/asciidoc/accessories/display/display_intro.adoc:18]: This image needs reshot by Brian. He has an Asana task to reshoot all (most) of the existing documentation imagery. So no need to nudge him about this one.

File: documentation/asciidoc/accessories/display/display_intro.adoc [hunk 2]
```diff
@@ -11,13 +11,28 @@ This whitepaper is an introduction to writing a Display Serial Interface (DSI) d
 
 The DSI display is designed to work with all models of Raspberry Pi, however early models that do not have mounting holes (the Raspberry Pi 1 Model A and B) will require additional mounting hardware to fit the HAT-dimensioned stand-offs on the display PCB.
 
-=== Physical Installation
+=== Connecting a Raspberry Pi
 
-The following image shows how to attach the Raspberry Pi to the back of the Touch Display (if required), and how to connect both the data (ribbon cable) and power (red/black wires) from the Raspberry Pi to the display. If you are not attaching the Raspberry Pi to the back of the display, take extra care when attaching the ribbon cable to ensure it is the correct way round. The black and red power wires should be attached to the GND and 5v pins respectively.
+The following image shows how to attach a Raspberry Pi to the back of the Touch Display using the included standoffs. It also shows the connections you need to make between Raspberry Pi and the display's PCB. You can also mount the touch display in a separate chassis if you have one available. The connections remain the same, though you may need longer cables depending on the chassis you use.
```
Comment (aallan) [inline documentation/asciidoc/accessories/display/display_intro.adoc:16]: I think this needs properly expanding with several images that show this process step-by-step. Possibly we should go down the route of an animated GIF, e.g. see https://www.raspberrypi.com/documentation/accessories/build-hat.html#preparing-your-build-hat.

File: documentation/asciidoc/accessories/display/display_intro.adoc [hunk 3]
```diff
@@ -11,13 +11,28 @@ This whitepaper is an introduction to writing a Display Serial Interface (DSI) d
...
 
-image::images/GPIO_power-500x333.jpg[DSI Display Connections]
+image::images/GPIO_power-500x333.jpg[DSI Display pinout]
 
-The other three pins should be left disconnected, unless connecting the display to an original Raspberry Pi 1 Model A or B. See the section on xref:display.adoc#legacy-support[legacy support] for more information on connecting the display to an original Raspberry Pi.
+Connect one end of the ribbon cable to the `RPI-DISPLAY` port on the Touch Display PCB (make sure the blue stripe faces the PCB) and then connect the other end to the `DISPLAY` port on the Raspberry Pi (the blue stripe should face outwards).
+
+==== Powering the Touch Display
+
+There are two options to power the Touch Display: using the Raspberry Pi's GPIO to send power, or powering the display directly with a separate micro USB cable.
```
Comment (aallan) [inline documentation/asciidoc/accessories/display/display_intro.adoc:24]: Should sub-section, if there are two options there should be two separate sections.

**Review by aallan — CHANGES_REQUESTED** [review]

File: documentation/asciidoc/accessories/display/display_intro.adoc [hunk 4]
```diff
@@ -11,13 +11,28 @@ This whitepaper is an introduction to writing a Display Serial Interface (DSI) d
...
+There are two options to power the Touch Display: using the Raspberry Pi's GPIO to send power, or powering the display directly with a separate micro USB cable. 
+
+WARNING: If you use a micro USB cable to power the display it must be mounted inside a chassis to reduce the risk of electrocution. The chassis also must block access to the display's PCB while its in use.
+
+The following instructions are only relevant if you're using the Raspberry Pi for power. Do not make these connections when powering the display through its micro USB port.
+
+Make sure the Raspberry Pi is powered off and unplugged, then connect one end of the black jumper wire to pin six (GND) on the Raspberry Pi and one end of the red jumper wire to pin four (5V). Next, connect the other end of the black wire to the GND pin on the display and the other end of the red wire to the 5V pin on the display.
+
+.Connect the red and black wires to 5V and GND
+image::images/touch-GPIO.png[Illustration of display pins, width=250]
```
Comment (aallan) [inline documentation/asciidoc/accessories/display/display_intro.adoc:33]: Don't use pixel widths, e.g. `width=250`. Instead use percentiles, e.g. `width=50%`

**Comment by aallan** [comment]
It's a decent start, but was anticipating a bit more of an overhaul here. I'm not the greatest fan of FAQ-based documentation, I think it's a real failure of narrative. This page has both a "troubleshooting" and "tips and tricks" section. These really need to get integrated into the documentation proper.

One thing to bear in mind is that this page has been fairly extensively modified on the `bookworm` branch in 82d55309dc9c6365259e77b6b91ff89ec8c11d34 by @lurch. I think the best move here would be to  change the base branch you're targeting the PR against and commit onto the `bookworm` branch.

**Comment by aallan** [comment]
Flipped it onto `bookworm` and it looks like there are some merge conflicts as a result.

**Comment by jehlers42** [comment]
Closing this due to messing up during conflict resolution. New PR for redone branch is #3121; comments are being taken into account.

**Pull request closed without merging**